src/algorithms/tracking/TrackSeeding.cc

```diff
@@ -4,6 +4,7 @@
 
 #include "TrackSeeding.h"
 
+#include "TVector2.h"
```

Comment:
This is a super nitpicky and minor comment, but Acts has a Vector2 object that uses eigen as the backend that is already included in this file in `Acts/Definitions/Algebra.hpp`. We could use that instead of TVector2 which would 
1. Reduce the number of includes
2. use eigen as the backend for matrix math, which while it probably doesn't make a difference in CPU speed for a simple dot product at least keeps us consistent